assets/less/main.less

```diff
@@ -0,0 +1,84 @@
+.day {
+  /* Sticky bottom navbar style */
+  .navbar-footer { background-color: #f5f5f5; }
+  .navbar-footer a { color: #1A5072; }
+
+  /* Fish info table */
+  .fish-info {
+    border: 2px solid #1a242f !important;
```

Comment:
Could any of these `!important`s be removed now that there is a more specific selector?